docs/management/admin/event-filters.asciidoc

```diff
@@ -4,6 +4,8 @@
 
 Event filters allow you to filter endpoint events that you do not need or want stored in {es} -- for example, high-volume events. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
 
+Event filters do not lower CPU usage on hosts; {elastic-endpoint} still monitors event data to detect and prevent possible threats, but the event data just isn't being written to {es}. To compare event filters with other endpoint artifacts, refer to <<endpoint-artifacts>>.
+
 NOTE: You must have the built-in `superuser` role to access this feature. For more information, refer to {ref}/built-in-users.html[Built-in users].
 
 IMPORTANT: Since an event filter blocks an event from streaming to {es}, be conscious of event filter conditions you set and any existing rule conditions. If there is too much overlap, the rule may run less frequently than specified and, therefore, will not trigger the corresponding alert for that rule. This is the expected behavior of event filters.
```

Comment:
```suggestion
IMPORTANT: Since an event filter blocks events from streaming to {es}, be conscious of event filter conditions you set and any existing rule conditions. If there is too much overlap, the rule may run less frequently than specified and therefore not trigger the corresponding alert for that rule. This is the expected behavior of event filters.
```